admin/attributes_controller.php

```diff
@@ -194,15 +194,19 @@
       break;
     case 'add_product_attributes':
       $current_image_name = '';
+      if (!isset($_POST['options_id']) || 
+           !isset($_POST['products_id']) || 
+           !empty($_POST['values_id'])) {
+           break;
+      }
```

Comment:
- `isset` can take multiple vars as comma-separated parameters.
- Shouldn't your `!empty()` be `empty()` ?
```suggestion
      if (!isset($_POST['options_id'], $_POST['products_id']) || empty($_POST['values_id'])) {
           break;
      }
```